core/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -1300,8 +1302,9 @@ public void release() {
 
     /**
      * Method to close the engine while the write lock is held.
+     * Must decrement {@param closedLatch} when closing work is done
```

Comment:
@dakrone aye. I had a commit locally fixing this which I forgot to push. Thanks for noticing.